CMakeLists.txt

```diff
@@ -87,37 +81,37 @@ include (cmake/ElektraPackaging.cmake)
 
 # ensure KDB_DB_SYSTEM is an absolute path
 if (NOT IS_ABSOLUTE "${KDB_DB_SYSTEM}")
-	set (KDB_DB_SYSTEM "${CMAKE_INSTALL_PREFIX}/${KDB_DB_SYSTEM}")
+        set (KDB_DB_SYSTEM "${CMAKE_INSTALL_PREFIX}/${KDB_DB_SYSTEM}")
 endif ()
 
 # process KDB_DB_SPEC so it is available everywhere
 if (NOT IS_ABSOLUTE "${KDB_DB_SPEC}")
-	set (KDB_DB_SPEC "${CMAKE_INSTALL_PREFIX}/${KDB_DB_SPEC}")
+        set (KDB_DB_SPEC "${CMAKE_INSTALL_PREFIX}/${KDB_DB_SPEC}")
 endif ()
 
 # now add all directories
 add_subdirectory (cmake)
 
 if (BUILD_DOCUMENTATION)
-	add_subdirectory (doc)
+        add_subdirectory (doc)
 endif (BUILD_DOCUMENTATION)
 
 # add scripts after doc
 add_subdirectory (scripts)
 
 # is there anything to build except documentation?
 if (BUILD_FULL OR BUILD_STATIC OR BUILD_SHARED)
-	if (BUILD_TESTING)
-		find_package (GTest)
-	endif (BUILD_TESTING)
+        if (BUILD_TESTING)
+                find_package (GTest)
+        endif (BUILD_TESTING)
 
-	add_subdirectory (src)
-	add_subdirectory (examples)
-	add_subdirectory (benchmarks)
+        add_subdirectory (src)
```

Comment:
It looks like your editor converted tabs to spaces or you used a wrong version of cmake-format. Please format with cmake-format as described in https://master.libelektra.org/doc/CODING.md